flow_post_lda.scala

```diff
@@ -1,270 +1,543 @@
-    val sip = row(8)
-    val dip = row(9)
-    val dport = row(10).toDouble
-    val sport = row(11).toDouble
-    val ipkt_bin = row(29).toDouble
-    val ibyt_bin = row(28).toDouble
-    val time_bin = row(30).toDouble
-    var p_case = 0.0
-
-    var ip_pair = row(9) + " " + row(8)
-    if (sip < dip & sip != 0) { ip_pair = row(8) + " " + row(9) }
-    
-    if ((dport <= 1024 | sport<=1024) & (dport > 1024 | sport>1024) & min(dport,sport) != 0){
-        p_case = 2
-        word_port = min(dport, sport)
-    }else if (dport >1024 & sport>1024){
-        p_case = 3
-        word_port = 333333
-    }else if (dport == 0 & sport != 0){
-        word_port = sport
-        p_case = 4
-    }else if (sport == 0 & dport != 0){
-        word_port = dport
-        p_case = 4
-    }else {
-        p_case = 1
-        if (min(dport, sport) == 0) {word_port = max(dport, sport)
-        }else { word_port = 111111}
-    }
-    //val word = ipkt_bin -1 + (ibyt_bin-1)*10 + (time_bin - 1)*100 + word_port*1000
-    val word = word_port.toString+"_"+time_bin.toString+"_"+ibyt_bin.toString+"_"+ipkt_bin.toString
-    var src_word = word
-    var dest_word = word
-    
-    if (p_case == 2 & dport < sport){ dest_word = "-1_"+dest_word
-    }else if (p_case == 2 & sport < dport){  src_word = "-1_"+src_word
-    }else if (p_case == 4 & dport == 0){  src_word = "-1_"+src_word
-    }else if (p_case == 4 & sport ==0){ dest_word = "-1_"+dest_word }
-    
-    row :+ word_port.toString :+ ip_pair :+ src_word.toString :+ dest_word.toString
-}
-
-//s1 = adjust_port(s1)
-val data_with_words = binned_data.map(row => adjust_port(row))
-
-
-val src_scored = data_with_words.map(row => {
-	val topic_mix_1 = topics.value.getOrElse(row(8),Array(0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05) ).asInstanceOf[Array[Double]]
-	val word_prob_1 = words.value.getOrElse(row(33),Array(0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05) ).asInstanceOf[Array[Double]]
-	val topic_mix_2 = topics.value.getOrElse(row(9),Array(0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05) ).asInstanceOf[Array[Double]]
-	val word_prob_2 = words.value.getOrElse(row(34),Array(0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05) ).asInstanceOf[Array[Double]]
-        var src_score = 0.0 
-        var dest_score = 0.0 
-	for ( i <- 0 to 19) {
-		 src_score += topic_mix_1(i) * word_prob_1(i)
-		 dest_score += topic_mix_2(i) * word_prob_2(i)
-      	}
-	 (min(src_score, dest_score), row :+ src_score :+ dest_score)
-	})
-
-
-//src_scored.take(10)
-
-
-var scored = src_scored.filter(elem => elem._1 < threshold).sortByKey().map( row => row._2.mkString(",") )
-
-scored.persist(StorageLevel.MEMORY_AND_DISK)
-scored.saveAsTextFile(scored_output_file)
-
-
-System.exit(0)
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+<<<<<<< HEAD
```

Comment:
I'm just looking at this. It happened merging 1.0.1 into intelml. Need to re-do this merge sorry. 
I'm going to take back last commit and try merge again. 
So sorry.